Sources/JavaScriptKit/BasicObjects/JSArray.swift

```diff
@@ -1,10 +1,10 @@
 /// A wrapper around [the JavaScript Array class](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array)
 /// that exposes its properties in a type-safe and Swifty way.
 public class JSArray: JSBridgedClass {
-    public static let constructor = JSObject.global.Array.function!
+    public static let constructor = JSObject.global.Array.function
```

Comment:
For types like these where the global must be available, how do you feel about declaring `constructor` as `JSFunction!`?

Reply:
I tried this, but then you have to add an explicit `JSFunction?` type signature on top, which means you still have to unwrap with `!` or `?` wherever this `constructor` is used. So after all, there's no benefit of unwrapping it and then wrapping back again into `JSFunction?` to satisfy the protocol requirement.